### ISSUE TITLE
Email activation link does not work

### ISSUE THREAD

**Issue by slightlychilled** [croogo/croogo#960]
When registering as a user, the email activation link that gets sent does not activate the user. When you click on the link, you will see a message saying account activated, but the status of the user is not changed to active. The function is not changing the status

```php
  public function activate($username, $activationKey)
    {
        // Get the user with the activation key from the database
        $user = $this->Users
            ->find()
            ->where([
                'username' => $username,
                'activation_key' => $activationKey
            ])
            ->first();
        if (!$user) {
            $this->Flash->error(__d('croogo', 'Could not activate your account.'));

            return $this->redirect(['action' => 'login']);
        }

        // Activate the user
        $user = $this->Users->activate($user);
        if (!$user) {
            $this->Flash->error(__d('croogo', 'Could not activate your account'));

            return $this->redirect(['action' => 'login']);
        }

        $this->Flash->success(__d('croogo', 'Account activated successfully.'));

        return $this->redirect(['action' => 'login']);
    } 
```

**Comment by rchavik** [comment]
Looks like incomplete `UsersTable::activate()`

**Comment by slightlychilled** [comment]
Could you expand?

**Comment by rchavik** [comment]
Fixed by 47de8a6caa0f7879d118226a86ee62048d40ea53
